src/style.css

```diff
@@ -2,0 +1,97 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: local('Avenir');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: local('Avenir-Book');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: local('Avenir-Heavy');
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.page__body {
+  margin-top: 20px;
+}
+
+.page__search-form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 20px;
+  padding: 0;
+}
+
+.page__search-form:nth-child(1) {
+  margin-top: 0;
+}
+
+.search {
+  box-sizing: border-box;
+  background-image: url(images/Search.svg);
```

Comment:
Ensure that the image path 'images/Search.svg' is correct and that the image exists in the specified directory.